frontend/src/scenes/feature-flags/FeatureFlagProjects.tsx

```diff
@@ -0,0 +1,115 @@
+import { useActions, useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { Field } from 'lib/forms/Field'
+import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
+import { featureFlagLogic } from './featureFlagLogic'
+import { organizationLogic } from '../organizationLogic'
+
+const records: Record<string, string>[] = [{ project: 'project-1-staging', flag_status: 'enabled' }]
+
+export function CopyFeatureFlagModal({
+    overlayRef,
+    contentRef,
+}: Pick<LemonModalProps, 'overlayRef' | 'contentRef'>): JSX.Element {
+    const { featureFlag, isProjectsModalOpen } = useValues(featureFlagLogic)
+    const { currentOrganization } = useValues(organizationLogic)
+    const { closeProjectsModal } = useActions(featureFlagLogic)
+
+    return (
+        <LemonModal
+            overlayRef={overlayRef}
+            contentRef={contentRef}
+            isOpen={isProjectsModalOpen}
+            onClose={closeProjectsModal}
+            title="Feature Flag copy"
+            description="Copy your flag and its configuration to another project"
+            footer={
+                <div className="flex-1 flex items-center justify-between">
+                    <div className="flex items-center gap-2" />
+                    <div className="flex items-center gap-2">
+                        <LemonButton form="ff-copy-modal-form" type="secondary" onClick={closeProjectsModal}>
+                            Cancel
+                        </LemonButton>
+                        <LemonButton htmlType="submit" type="primary">
+                            Copy
+                        </LemonButton>
+                    </div>
+                </div>
+            }
+        >
+            <div className="max-w-120">
+                <Form
+                    logic={featureFlagLogic}
+                    formKey="ffCopyModal"
+                    id="ff-copy-modal-form"
+                    enableFormOnSubmit
+                    className="space-y-4"
+                >
+                    <div className="flex gap-2">
+                        <div className="flex gap-2 flex-1 flex-col">
+                            <div className="font-semibold leading-6 h-6">Feature Flag key</div>
+                            <div className="border px-3 rounded h-10 text-center flex items-center justify-center max-w-200">
+                                <span className="font-semibold truncate">{featureFlag.key}</span>
+                            </div>
+                        </div>
+                        <div className="flex gap-2 flex-1 flex-col items-center max-w-10">
+                            <div className="h-6" />
+                            <IconArrowRight className="h-10" fontSize="30" />
+                        </div>
+                        <Field name="Project" label="Project" className="flex-1">
+                            <LemonSelect
+                                value={currentOrganization?.teams?.[0].id}
```

Comment:
is this the current team?

Reply:
yes, this is just a placeholder for now. The whole tab will be hidden for anyone having only 1 team and here, some other team than the current team can be pre-selected (or none)